yamux/src/connection.rs

```diff
@@ -821,6 +909,18 @@ impl<T: AsyncRead + AsyncWrite + Unpin> Active<T> {
         Action::None
     }
 
+    fn make_new_stream(&mut self, id: StreamId, window: u32, credit: u32) -> Stream {
+        let config = self.config.clone();
+
+        let (sender, receiver) = mpsc::channel(10);
```

Comment:
I am assuming that this number is chosen arbitrarily, correct? If so, mind documenting that fact? Would allow us to tune it in the future without wondering why `10` was chosen.